roles/create_rds_global_cluster/tasks/create.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Run 'create_rds_global_cluster' role create operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Create rds global database
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"
+        engine: "{{ create_rds_global_cluster_engine  | default('aurora-postgresql')}}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
```

Comment:
Isn't engine_version mandatory, when there is an engine parameter? Can it be 'None' (omit) ?

Reply:
fixed.